README.md

```diff
@@ -127,9 +127,9 @@ image files.
 You can show the images on a web page [1].  Those made with other realistic
 workloads are also available [2,3,4].
 
-[1] https://damonitor.github.io/doc/html/latest/admin-guide/mm/damon/start.html#visualizing-recorded-patterns  
-[2] https://damonitor.github.io/test/result/visual/latest/rec.heatmap.1.png.html  
-[3] https://damonitor.github.io/test/result/visual/latest/rec.wss_sz.png.html  
+[1] https://damonitor.github.io/doc/html/latest/admin-guide/mm/damon/start.html#visualizing-recorded-patterns
+[2] https://damonitor.github.io/test/result/visual/latest/rec.heatmap.1.png.html
+[3] https://damonitor.github.io/test/result/visual/latest/rec.wss_sz.png.html
```

Comment:
These are intended trailing whitespaces, to let the lines be separated.  I think this should be excluded from the tool?

Reply:
How about adding `<br>` instead of whitespaces? I've updated it using `<br>` instead. Please let me know if you don't like this.

Reply:
Do you mean adding one more blank line between the lines?  I'm not very familiar with markdown, but it would result in treating the lines as paragraphs, and therefore add some gaps between the lines.

Meanwhile, I found this PR, which seems could work for this case:
https://github.com/pre-commit/pre-commit-hooks/pull/58

That is, maybe we could add `--markdown-linebreak-ext=md` to the configuration?